src/nvim/eval.c

```diff
@@ -8470,9 +8546,15 @@ handle_subscript(
       } else {
         s = (char_u *)"";
       }
-      ret = get_func_tv(s, lua ? slen : -1, rettv, (char_u **)arg,
-                        curwin->w_cursor.lnum, curwin->w_cursor.lnum,
-                        &len, evaluate, pt, selfdict);
+
+      funcexe_T funcexe;
+      memset(&funcexe, 0, sizeof(funcexe));
```

Comment:
Instead of memset, i would suggest defining `FUNCEXE_INIT`  in analogy with existing types.